.goreleaser.yml

```diff
@@ -0,0 +1,21 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+  - go mod download
+builds:
+- env:
+  - CGO_ENABLED=0
+  goarch:
+  - amd64
+  goos:
+  - linux
+  - darwin
+dockers:
+- image_templates:
+  - "caarlos0/openvpn-exporter:latest"
```

Comment:
Remove references to personal repo

Reply:
Sorry, don’t know where you want to push the images to...  what’s the Docker hub account you want the images to be pushed to?